sequence_transform.c

```diff
@@ -243,6 +255,45 @@ void log_rule(st_trie_t *trie, st_trie_search_result_t *res) {
     // Terminator
     uprintf("\n");
 }
+//////////////////////////////////////////////////////////////////////
+__attribute__((weak)) void sequence_transform_on_missed_rule_user(const st_trie_rule_t *rule)
+{
+    uprintf("Missed rule! %s -> %s\n", rule->sequence, rule->transform);
+}
+//////////////////////////////////////////////////////////////////////
+void st_find_missed_rule(void)
+{
+    char sequence_str[SEQUENCE_MAX_LENGTH + 1] = {0};
+    char transform_str[TRANSFORM_MAX_LEN + 1] = {0};
+    static int search_len_start = 1;
+    // Start search from last space
+    if (PREV_KEY == KC_SPACE) {
+        search_len_start = key_buffer.context_len;
+        return;
+    }
+    // Buffer starts rolling when full, so dec search search_len_start.
+    if (key_buffer.context_len == key_buffer.size) {
+        search_len_start = st_max(1, search_len_start - 1);
+    } else {
+        // Don't let search_len_start get ahead of context len.
+        // (this handles backspace and buffer reset)
+        search_len_start = st_min(search_len_start, key_buffer.context_len - 1);
+    }
+    st_trie_rule_t result;
+    result.sequence = sequence_str;
+    result.transform = transform_str;
+    int t = timer_read32();
+    const uint8_t next_start = st_trie_get_rule(&trie, &key_buffer, search_len_start, &result);    
+#ifdef SEQUENCE_TRANSFORM_LOG_GENERAL
+    t = timer_elapsed32(t);
```

Comment:
if log_general is not defined then it cant compile - `unused variable 't'`

Reply:
Good catch. Will fix.